app/src/main/res/layout/content_tab_switcher.xml

```diff
@@ -13,22 +13,69 @@
   ~ See the License for the specific language governing permissions and
   ~ limitations under the License.
   -->
-<androidx.recyclerview.widget.RecyclerView xmlns:android="http://schemas.android.com/apk/res/android"
+<LinearLayout
+    xmlns:android="http://schemas.android.com/apk/res/android"
     xmlns:app="http://schemas.android.com/apk/res-auto"
     xmlns:tools="http://schemas.android.com/tools"
-    android:id="@+id/tabsRecycler"
     android:layout_width="0dp"
     android:layout_height="0dp"
-    android:layout_marginStart="7dp"
-    android:layout_marginEnd="7dp"
-    android:clipToPadding="false"
-    android:paddingTop="10dp"
-    app:layoutManager="androidx.recyclerview.widget.GridLayoutManager"
+    android:animateLayoutChanges="true"
+    android:orientation="vertical"
     app:layout_constraintBottom_toBottomOf="parent"
     app:layout_constraintEnd_toEndOf="parent"
     app:layout_constraintStart_toStartOf="parent"
-    app:layout_constraintTop_toBottomOf="@id/appBarLayout"
-    tools:itemCount="3"
-    tools:listitem="@layout/item_tab"
-    tools:showIn="@layout/activity_tab_switcher"
-    tools:spanCount="2" />
\ No newline at end of file
+    app:layout_constraintTop_toBottomOf="@id/appBarLayout">
+
+    <androidx.cardview.widget.CardView
+        android:id="@+id/tabFeatureAnnouncement"
+        android:visibility="gone"
+        app:cardElevation="4dp"
+        android:layout_width="match_parent"
+        android:layout_height="wrap_content">
+
+        <LinearLayout
+            android:layout_width="match_parent"
+            android:layout_height="wrap_content"
+            android:orientation="horizontal">
+
+            <com.duckduckgo.common.ui.view.text.DaxTextView
+                android:layout_width="0dp"
+                android:layout_height="wrap_content"
+                android:layout_weight="1"
+                android:layout_margin="16dp"
```

Comment:
You can use existing `dimens` for this layout. 
See https://github.com/duckduckgo/Android/blob/develop/common/common-ui/src/main/res/values/design-system-dimensions.xml